scripts/test_all.wren

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env wrenc
+
+import "io" for Directory, File
+import "os" for Process
+import "meta" for Meta
+import "essentials" for Strings
+
+// 1. set cwd to wren-testie root
+
+// TODO (enhancement): static method File.tempfile()
+// TODO (enhancement): reading stdout of a process
+
+var tmpfile = ".git.root.%(Process.pid)"
+Process.exec("sh", ["-c", "git rev-parse --show-toplevel > %(tmpfile)"])
+var rootdir = File.read(tmpfile).trim()
+File.delete(tmpfile)
```

Comment:
Is all this because you don't often work out of the root directory of projects?  Many of my utility scripts ASSUME this... this isn't bad, but it's also not entirely portable...

Can we get the absolute pathing given argument 0, the script name itself?  Or is that stripped of path info?

Reply:
It's not exactly public but you have the `Path` class at your disposal as well for navigation around a path string.

Reply:
```bash
$ cd tests
$ cat > path.wren
import "os" for Process
System.print(Process.arguments)
System.print(Process.allArguments)
System.print(Process.cwd)
^D
$ wrenc ./path.wren
[]
[wrenc, ./path.wren]
/Users/glennjackman_1/src/other/wren/joshgoebel/wren-testie/tests
```
It's just because the error message for running a test file is so maddening _unless_ you're at the repo root
```bash
$ wrenc ./expect.wren
Could not load module 'testie'.
at (script) (././expect.wren line 1)
```
I haven't really gotten the module resolver. I thought imports of relative paths are relative to the directory of the file currently being evaluated, and this is why `scripts/test_all.wren` has to import `"../testie"` -- but then that error message baffles me: relative to `tests/expect.wren`, testie is in `../`

Reply:
> I haven't really gotten the module resolver. I thought imports of relative paths are relative to the directory of the file currently being evaluated,

Correct.  And the **original** script (first argument to `wrenc`) root is used for walking up the directory to find `wren_modules` folders.

Pretty sure there is a bug in `Path` class here:

- `./../testie` normalizes to `testie`, which is wrong.

PR forthcoming.

Reply:
https://github.com/joshgoebel/wren-console/pull/14

Reply:
Don't assume we're bug free. :)  Might spend lots of time writing code to get around bugs that are much simpler to fix than the code you're writing to avoid them. :)

Reply:
I'm thinking I should remove the check for git rootdir, and maybe just replace it with a comment.

Reply:
You try my PR and see if it works fine without?

Reply:
It seems to work fine, but because I have to list a relative dir, I'm pretty much stuck at forcing execution from the project root.

Reply:
Huh?